infrastructure/kubernetes/workflows-default/daskkubernetes-role.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: daskkubernetes
+rules:
+  - apiGroups:
+      - ""  # indicates the core API group
+    resources:
+      - "pods"
+    verbs:
+      - "get"
+      - "list"
+      - "watch"
+      - "create"
+      - "delete"
+  - apiGroups:
+      - ""  # indicates the core API group
```

Comment:
should there be anything in this category?

Reply:
Good question but, I think this is good. This `""` indicates that the permissions are to the core k8s API. This role is giving some extremely powerful and dangerous abilities, at the heart of the k8s.